opm/grid/common/ZoltanPartition.cpp

```diff
@@ -279,12 +280,11 @@ class ZoltanSerialPartitioner
             for (int i = 0; i < numExport; ++i) {
                 ++numberOfExportedVerticesPerProcess[exportToPart[i]];
             }
-            int dummyForRoot = 0;
-            cc.scatter<int>(numberOfExportedVerticesPerProcess.data(), &dummyForRoot, 1, root);
-        } else {
-            cc.scatter<int>(nullptr, &numImport, 1, root);
         }
 
+        cc.scatter(numberOfExportedVerticesPerProcess.data(), &numImport, 1, root);
```

Comment:
For an empty vector, data() may or may not return a null pointer. Will scatter() work correctly here even if a non-null pointer is passed? Also for scatterv() on line 332-333.